server/src/main/java/com/cloud/resource/RollingMaintenanceManagerImpl.java

```diff
@@ -571,7 +571,7 @@ private void performPreFlightChecks(List<Host> hosts, int timeout, String payloa
         for (Host host : hosts) {
             Ternary<Boolean, String, Boolean> result = performStageOnHost(host, Stage.PreFlight, timeout, payload, forced);
             if (result.third() && !hostsToAvoidMaintenance.containsKey(host.getId())) {
-                s_logger.debug("Host " + host.getId() + " added to the avoid maintenance set");
+                s_logger.debug(String.format("%s added to the avoid maintenance set", host));
                 hostsToAvoidMaintenance.put(host.getId(), "Pre-flight stage set to avoid maintenance");
             }
```

Comment:
This code seems repeated, but changing parameters; We could extract it to a method.

Reply:
Done!